src/Microsoft.SqlTools.ServiceLayer/Scripting/SmoScriptingOperation.cs

```diff
@@ -79,6 +79,7 @@ protected string GetServerNameFromLiveInstance(string connectionString, string a
             string serverName = null;
             using (SqlConnection connection = new SqlConnection(connectionString))
             {
+                connection.RetryLogicProvider = Connection.ReliableConnection.RetryPolicyUtils.ServerlessWaitRetryLogicProvider();
```

Comment:
If you ask me, the choice of name for this this class is weird. I did not see anything in there that suggested "Serverless" stuff. But what do I know? :)